src/app/app-routing.module.ts

```diff
@@ -3,8 +3,8 @@ import { RouterModule, Routes } from '@angular/router';
 import { AppComponent } from './app.component';
 import { IndexComponent } from './components/index/index.component';
 import { ItinerarylistComponent } from './components/itinerary/itinerarylist/itinerarylist.component';
+import { MyItinerariesComponent } from './components/itinerary/my-itineraries/my-itineraries.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181827)
```suggestion
import { MyItinerariesComponent } from "./components/itinerary/my-itineraries/my-itineraries.component";
```